homeassistant/components/light/mochad.py

```diff
@@ -0,0 +1,94 @@
+"""
+Contains functionality to use a X10 dimmer over Mochad.
+
+For more details about this platform, please refer to the documentation at
+https://home.assistant.io/components/light.mochad
+"""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
+from homeassistant.components import mochad
+from homeassistant.const import (CONF_NAME, CONF_PLATFORM)
+from homeassistant.helpers import config_validation as cv
+
+DEPENDENCIES = ['mochad']
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ADDRESS = 'address'
+CONF_DEVICES = 'devices'
+
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_PLATFORM): mochad.DOMAIN,
+    CONF_DEVICES: [{
+        vol.Optional(CONF_NAME): cv.string,
+        vol.Required(CONF_ADDRESS): cv.x10_address,
+        vol.Optional('comm_type'): cv.string,
```

Comment:
Done, I also updated the mochad switch component which has the same constant. I did the same for address which will also be common between all mochad components.